src/studio/src/designer/backend/Services/Implementation/SchemaModelService.cs

```diff
@@ -114,6 +114,10 @@ public async Task<(string, string)> CreateSchemaFromTemplate(string org, string
         {
             var altinnGitRepository = _altinnGitRepositoryFactory.GetAltinnGitRepository(org, repository, developer);
 
+            // Even if we have a default value of empty string the parameter is null if not provided
+            // hence we check and set it to and empty string if it's null.
```

Comment:
The comment is misleading. The mechanism for default value is used only if the parameter is **omitted**. If a caller actively set the parameter to null, the "value" will be null.

Reply:
The caller did not set the value.